classes/controller/AdminController.php

```diff
@@ -2067,7 +2067,8 @@ public function initHeader()
                 'employee' => $this->context->employee,
                 'search_type' => Tools::getValue('bo_search_type'),
                 'bo_query' => Tools::safeOutput(Tools::getValue('bo_query')),
-                'quick_access' => empty($quick_access) ? false : $quick_access,
+//                'quick_access' => empty($quick_access) ? [] : $quick_access, // new
+                'quick_access' => empty($quick_access) ? false : $quick_access, // old
```

Comment:
Mistake?

Reply:
There is a lot of fail jobs i'm testing thing to see what make the tests go red.